.github/workflows/companion-deploy.yml

```
@@ -4,8 +4,8 @@ name: Companion Deploy
 on:
   push:
     branches: [main]
-    paths:
-    - 'packages/@uppy/companion/**'
+    tags:
+      - '@uppy/companion*'
```

We don't want it to match `@uppy/companion-client` releases ;)
```suggestion
      - '@uppy/companion@*'
```